public-ui/src/app/highlight-search/highlight-search.component.html

```diff
@@ -0,0 +1,14 @@
+<span *ngIf="isSource"><b> Source </b></span>
+<span *ngIf="isStandard"><b> Standard </b></span>
+<span *ngIf="indNum">{{indNum}}. </span>
+<span *ngIf="searchTerm; else noSearchTerm">
```

Comment:
spans usually don't contain spans - I think ng-container might be more appropriate here

Reply:
Done. Thanks. Did not know this.